packages/acceptance-tests/pkg-tests-specs/sources/node-modules.test.ts

```diff
@@ -713,4 +713,31 @@ describe(`Node_Modules`, () => {
       },
     )
   );
+
+  test(`should survive interrupted install`,
+    makeTemporaryEnv(
+      {
+        private: true,
+        workspaces: [`foo`],
+      },
+      {
+        nodeLinker: `node-modules`,
+      },
+      async ({path, run, source}) => {
+        await writeJson(npath.toPortablePath(`${path}/foo/package.json`), {
+          name: `foo`,
+          dependencies: {
+            'has-bin-entries': `1.0.0`,
+          },
+        });
+
+        await run(`install`);
+
+        // Simulate interrupted install
+        await xfs.removePromise(`${path}/node_modules/has-bin-entries` as PortablePath);
+
+        await run(`add`, `create-test-app`);
```

Comment:
I think you should add at least two other `expect` to make sure that `has-been-entries` has been correctly regenerated (for example by checking that its `package.json` and `index.js` are there)

Reply:
Done